lib/agent/ocsp_response_cache.js

```diff
@@ -320,6 +317,93 @@ function OcspResponseCache()
       return {err: Errors.createOCSPError(ErrorCodes.ERR_OCSP_FAILED_PARSE_RESPONSE)};
     }
   }
+
+  function updateCache(jsonObject)
+  {
+    // Get the size of cache retrieved from the cache server
+    var responseCacheSize = Object.keys(jsonObject).length;
+
+    // Check if there are previous entries to append
+    if (prevCacheObj)
+    {
+      // Count overlap between previous cache and response cache
+      var cacheOverlapCount = 0;
+      for (let entry in jsonObject)
+      {
+        if (entryExists(prevCacheObj, entry))
+        {
+          cacheOverlapCount++;
```

Comment:
Is it possible to delete the entries in the older cache over here itself? This might save us the trouble of going through the entire older cache again at Line 351.

Reply:
Done